tests/page-layout-both/OPS/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>If a reflowable spine item follows a pre-paginated one, it should appear on the next page.</dc:description>
+    <dc:identifier id="pub-id">page-layout-both</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>page-layout-both</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-33/#property-layout-local</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#page-layout-both</meta>
+    <meta property="dcterms:modified">2021-01-01T00:00:00Z</meta>
```

Comment:
There is something odd going on. It seems that nothing is said about layout overrides (ie, [§5.2.1.1](https://www.w3.org/TR/epub-33/#layout-overrides) in the content document) in the RS. Is that an oversight or intentional? 

@mattgarrish ?

Reply:
This section is under-specified, which I assume leads to it's under-implementation.

Reply:
Probably because no reading system behaviours were specified for the overrides, so nothing got pulled out. They only had authoring requirements. See https://www.w3.org/publishing/epub3/epub-packages.html#layout-property-flow-overrides

I guess reading systems "may" override the respective spine item as declared, but we'd need to write new prose for this. Anything higher would invalidate RSes that don't support the properties, and support for FXL is always optional.

Reply:
Ok. I have opened a new issue for this (#1941), and mark the comment as resolved.